analytics_data_api/utils.py

```diff
@@ -31,3 +31,38 @@ def set_user_auth_token(user, key):
     Token.objects.create(user=user, key=key)
 
     print "Set API key for user %s to %s" % (user, key)
+
+
+def matches(answer_one, answer_two):
+    """ Check whether two answers match sufficiently for consolidation. """
+    return (
```

Comment:
@clintonb Do you want me to switch to backslashes instead?  I think parentheses are <a href="https://www.python.org/dev/peps/pep-0008/#maximum-line-length">preferred</a> for multiline expressions in pep8, and I can't think of another way to express this multiline conditional without parentheses.